### DISCUSSION TITLE
Filter for already have / approved

### DISCUSSION THREAD

**Discussion by Dragon470** [sct/overseerr#3385] [Feature Requests]
Filter so in discovery it removes all titles you already have or approved.

**Comment by Percy2Live** [comment]
And it would be great to have an option vice versa: Show only shows you already have (this gives you a nice overview about shows which have a new season available)

**Comment by jamesdthor** [comment]
Yes this would be great, clicking "Discover" ends up being a shopping list of items I already have and then having to cycle through various pages to find stuff I need. Would love to be able to toggle on and off stuff thats already there/approved!

**Comment by SebbeJohansson** [comment]
I agree on this. I feel like this is a huge downside to using overseerr.

**Comment by R0cketM0nster** [comment]
I would also like to vote for a filter option to hide movies/shows already in library please. I have to scroll through pages of movies/shows I already have to find anything new.

**Comment by dantebarba** [comment]
Same. Both filters should be easy to implement!
